src/cfb.rs

```diff
@@ -279,6 +279,7 @@ impl Sectors {
             Vec::new()
         };
         while sector_id != ENDOFCHAIN {
+            // FIXME: see tests/OOM_alloc3.xls, this call OOMs
```

Comment:
I think the changes in this commit should be merged with the next commit.

Reply:
> The separate commit adding the test case is useful to verify the abort before verifying the fix.

In general it is best not to commit any changes that break the build process or the test suite since it makes running `git bisect` harder. I've worked on projects where both of these are strictly forbidden. So the test case should be rolled into the commit that fixes it.